packages/salesforcedx-vscode-core/src/util/cliConfiguration.ts

```diff
@@ -53,13 +53,18 @@ export function disableCLITelemetry() {
 export async function isCLITelemetryAllowed(
   projectPath: string
 ): Promise<boolean> {
-  // if (isCLIInstalled()) {
-  //   const forceConfig = await new ForceConfigGet().getConfig(
-  //     projectPath,
-  //     SFDX_CONFIG_DISABLE_TELEMETRY
-  //   );
-  //   const disabledConfig = forceConfig.get(SFDX_CONFIG_DISABLE_TELEMETRY) || '';
-  //   return disabledConfig !== 'true';
-  // }
+  if (isCLIInstalled()) {
+    try {
+      const forceConfig = await new ForceConfigGet().getConfig(
+        projectPath,
+        SFDX_CONFIG_DISABLE_TELEMETRY
+      );
+      const disabledConfig =
+        forceConfig.get(SFDX_CONFIG_DISABLE_TELEMETRY) || '';
+      return disabledConfig !== 'true';
+    } catch (e) {
+      console.log('Error checking cli settings: ' + e);
+    }
+  }
   return true;
```

Comment:
Do we want to err on this side if an exception happens?

Reply:
@xyc That mimics the prior behavior. Do you feel it should be otherwise?

Reply:
Yes, especially since this default to true when a generic error happens, not only when the CLI version is earlier.